recipes.json

```diff
@@ -1,43 +1,109 @@
 {
     "carbonara": {
         "ingredients": {
-            "egg": 2,
-            "parmesan cheese": 1,
-            "spaghetti": 1,
-            "bacon": 4
+            "egg": [
+                2,
+                "unit"
+            ],
+            "parmesan cheese": [
+                200,
+                "Gramms"
```

Comment:
do we want this as the full word?

do we not want like 200, "g"?

Reply:
moving to just g should be fine